.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "extends": [
+    "lisk-base",
+    "airbnb-base"
+  ],
```

Comment:
Airbnb base includes  9 set of rules defined inside the package, 3 set of rules as dependencies and these all are on top of eslint base rules. from all these only the rules under `eslint-plugin-react` are react related which you can simply ignore them by not adding the configuration in your webpack configs. I guess all fine. unless there's another issue which I'm not aware of.